server/server.go

```diff
@@ -466,7 +466,7 @@ func New(config *Config) (*Server, error) {
 		return nil, err
 	}
 
-	imageService, err := storage.GetImageService(store, config.DefaultTransport, config.InsecureRegistries)
+	imageService, err := storage.GetImageService(store, config.DefaultTransport, config.InsecureRegistries, config.Registries)
```

Comment:
mmm they are unrelated, I guess it's personal taste, if you want I'll have `Registries` comes first